CONTRIBUTING.md

```diff
@@ -124,152 +126,199 @@ Assign the original repo to a remote called "upstream"
 > `npm config set color false`
 > Note that this change takes effect when a new command prompt window is opened; the current window will not be affected.
 
-Build a local copy of video.js and run tests
+#### Running tests
 
-```bash
-grunt dist
-grunt test
+To run the tests all you need to do is run
+
+```sh
+$ npm test
 ```
 
-Video.js is also configured to run tests with Karma. Karma is installed as a grunt plugin to run QUnit tests in real browsers, as opposed to simply running the tests in phantomjs, a headless browser. To run the tests with Karma:
+This will build video.js locally and run the tests using [Karma](https://karma-runner.github.io/1.0/index.html) which runs our tests in actual browsers.
+
+#### Building videojs
+
+To build video.js, simply run
 
-```bash
-grunt karma:dev
+```sh
+$ npm run build
 ```
 
-At this point you should have a built copy of video.js in a directory named `dist`, and all tests should be passing.
+This outputs an `es5/` and `dist/` folder. The `es5/` folder is used by bundling tools like browserify and webpack to package video.js into projects. The `dist/` folder has pre-compiled versions of video.js, including a minified version and the CSS file. This file can be included in page via a `<script></script>` tag.
 
-### Making Changes
 
-Whether you're adding something new, making something better, or fixing a bug, you'll first want to search the [GitHub issues](https://github.com/videojs/video.js/issues) and [plugins list](https://github.com/videojs/video.js/wiki/Plugins) to make sure you're aware of any previous discussion or work. If an unclaimed issue exists, claim it via a comment. If no issue exists for your change, submit one, follwing the [issue filing guidelines](#filing-issues).
+#### Testing Locally
+
+Besides running automated tests, you often want to run video.js manually and play around with things as you're developing. A few things are provided to make it easier.
 
-There are two categories of changes in video.js land, features and hotfixes (Video.js follows a branching model similar to [gitflow](http://nvie.com/posts/a-successful-git-branching-model/)). Hotfixes are for urgent fixes that need to be released immediately as a patch. Features are for everything else (including non-urgent fixes). If you think you have a hotfix scenario, verify that (via comment) before starting the work. We'll focus on features here, but you can swap `hotfix` for `feature` in any command.
+#### Sandbox test directory
 
-Start a new development branch
+There's a sandbox directory where you can add any file and it won't get tracked in git. To start you can copy the example index file and see a working version of a player (using the local source code) by loading it in a browser.
 
-```bash
-contrib feature start
+```sh
+$ cp sandbox/index.html.example sandbox/index.html
+$ open sandbox/index.html
```

Comment:
Maybe suggest running `npm start` and heading over to `localhost:9999/sandbox/`?